In [7]:
import pandas as pd
import matplotlib.pyplot as plt

In [8]:
df = pd.read_csv("/Users/kaanadaleti/Desktop/dsa proje/data/adidas_kanye_yeezy_trends.csv")
df.head()

,Time,Kanye West,ADIDAS,Yeezy
0,2022-08-31,6,45,1
1,2022-09-01,7,43,1
2,2022-09-02,12,44,1
3,2022-09-03,12,50,1
4,2022-09-04,13,53,1


In [9]:
print(df.shape)

(92, 4)


In [10]:
df.describe()
df.corr(numeric_only=True)

,Kanye West,ADIDAS,Yeezy
Kanye West,1.000000,0.038481,0.559794
ADIDAS,0.038481,1.000000,-0.036059
Yeezy,0.559794,-0.036059,1.000000


In [11]:
from scipy.stats import ttest_ind

# tarih sütununu datetime yap
df['Time'] = pd.to_datetime(df['Time'])

# event date
event_date = '2022-10-25'

before = df[df['Time'] < event_date]['ADIDAS']
after = df[df['Time'] >= event_date]['ADIDAS']

t_stat, p_value = ttest_ind(before, after)

print("T-stat:", t_stat)
print("P-value:", p_value)

T-stat: -4.234224045193987
P-value: 5.5209736341912384e-05


## Correlation Analysis

Correlation analysis reveals that search interest for Kanye West and Yeezy is moderately positively correlated (r ≈ 0.56).

In contrast, the correlation between Kanye West and Adidas is very weak, suggesting that attention does not strongly transfer to the brand.